pkgs/development/tools/poetry2nix/poetry2nix/overrides/default.nix

```diff
@@ -1078,9 +1078,9 @@ lib.composeManyExtensions [
 
           buildInputs = old.buildInputs or [ ] ++ [
             pkgs.which
-          ] ++ lib.optionals enableGhostscript [
+          ] ++ lib.optional enableGhostscript [
             pkgs.ghostscript
-          ] ++ lib.optionals stdenv.isDarwin [
+          ] ++ lib.optional stdenv.isDarwin [
```

Comment:
won't this result in a list of lists?

Reply:
Indeed it will, I've audited all overrides for the same problem and found a few more which have all been corrected.